e2e/cypress/tests/integration/collapsed_reply_threads/root_post_spec.js

```diff
@@ -84,4 +99,49 @@ describe('Collapsed Reply Threads', () => {
         // * There should be no thread item anymore
         cy.get('article.ThreadItem').should('have.lengthOf', 0);
     });
+
+    it('MM-T4446 delete single reply on thread (current behavior is incorrect, see comments)', () => {
```

Comment:
I suggest to match the title with Zephyr. So either update here or in Zephyr.
```suggestion
    it('MM-T4446 CRT - Delete single reply post on a thread', () => {
```